StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetLPMConfirmFlowTests.swift

```diff
@@ -53,7 +56,7 @@ final class PaymentSheet_LPM_ConfirmFlowTests: XCTestCase {
     }
 
     func testBancontactConfirmFlows() async throws {
-        try await _testConfirm(intentKinds: [.paymentIntent], currency: "EUR", paymentMethodType: .dynamic("bancontact")) { form in
+        try await _testConfirm(intentKinds: [.paymentIntent, .deferredPaymentIntent], currency: "EUR", paymentMethodType: .dynamic("bancontact")) { form in
```

Comment:
+1 to @yuki-stripe suggestion's around not adding deferred intent as it's own type, but rather just ignore it for only Blik in the code below.